app/src/main/java/com/pokemonshowdown/data/ItemDex.java

```diff
@@ -90,12 +54,14 @@ public String getItemName(String name) {
         return item;
```

Comment:
And there should be a null check for itemId input if you plan on doing String operation